### ISSUE TITLE
Rename 'aria-roles' rule as 'aria-valid-role-value'

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#770]
Since ```aria-roles``` rule fails for using invalid and abstract role names, I think it is better to rename it to ```aria-valid-role-value```, which would be more accurate and consistent with the current rule names

**Comment by dylanb** [comment]
This is a breaking change. Rule names are not intended to be human readable, therefore I am going to vote against this change.

**Comment by iamrafan** [comment]
Sure, np. It is not a blocker. Since we are going to have a new rule 'aria-allowed-role' and already have a rule called 'aria-valid-attr-value', I felt it would be a good idea to change the name

**Comment by WilcoFiers** [comment]
I quite like the suggestion. There are a few other rules that could do with a better ID. Things like "label".  We probably could've done this if it had come up a few months back, but we're really close to 3.0 now. We can't make a change like that without giving notice way in advance.

**Comment by iamrafan** [comment]
@WilcoFiers : Sure, I wasn't asking to add this in 3.0, I was suggesting it would be a good idea in the long run

**Comment by marcysutton** [comment]
I think we'd need an aliasing system in order for this to happen, regardless of a major release. There are too high of risks to custom rules and backwards compatibility.

**Comment by dylanb** [comment]
@marcysutton BIG THUMBS UP on that
